### DISCUSSION TITLE
Is there a user's manual for chart-fx?

### DISCUSSION THREAD

**Discussion by flyinghigh012** [GSI-CS-CO/chart-fx#505] [General]
Is there a user's manual for chart-fx?

In ScatterAndBubbleRendererSample.java, there is a DEMO_DATA_FILE being set to testdata/2017_OECD_data.csv, where can I get this file?

Also, I got one of the other sample plots working, but how can I grab the plot and move it around? It only zooms in on the area that you select.

**Comment by RalphSteinhagen** [comment]
@flyinghigh012 thanks for your interest in our little lib.

There isn't a user's manual per se, because we reckon that it would be outdated rather quickly, and thus opted for providing the minimum necessary info in the [README.md](https://github.com/GSI-CS-CO/chart-fx) and examples that both illustrate the capabilities as well as the user API.

The samples are located [here](https://github.com/GSI-CS-CO/chart-fx/tree/master/chartfx-samples/src/main/java/de/gsi) which also includes also the folder with the [testdata](https://github.com/GSI-CS-CO/chart-fx/tree/master/chartfx-samples/src/main/resources/de/gsi/chart/samples/testdata) you have been looking for.

To get started the easiest way is to compile and run the examples as described [here](https://github.com/GSI-CS-CO/chart-fx#examples), notably via the shell/Maven commands:
```bash
gh repo clone GSI-CS-CO/chart-fx
cd chart-fx/
mvn compile install
mvn exec:java
```
and you should get something like this:
![image](https://user-images.githubusercontent.com/46007894/162383628-6ea08c3a-decd-49eb-a084-0f0bab8f7db0.png)
We know it's basic, but sometimes simpler solutions are easier to read and to copy/paste/adopt...

Please let us know your use-case, feel free to share snaphshots of your application, and you are most welcome to contribute via PRs, discussions, etc. While we wrote the lib primarily for our internal use at FAIR, we are curious about how it is used elsewhere as this info helps us also to steer the further development of ChartFX.

Enjoy!

**Reply by flyinghigh012** [reply]
This was helpful, I was able to get the RunChartSamples up and running, thanks.
